build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -65,6 +65,63 @@
     "@llvm-project//mlir:TensorDialect": ["MLIRTensor"],
     "@llvm-project//mlir:NVVMDialect": ["MLIRNVVMIR"],
     "@llvm-project//mlir:ROCDLDialect": ["MLIRROCDLIR"],
+    # MHLO.
+    # TODO: Rework this upstream so that Bazel and CMake rules match up
```

Comment:
:open_mouth: wow this is awful

Reply:
Yeah, I didn't realize how awful until it was plainly visible like this.